### DISCUSSION TITLE
썸네일 이슈 관련하여 문의드립니다.

### DISCUSSION THREAD

**Discussion by baramoss** [pronist/hELLO#183] [Q&A] [answered]
안녕하세요. 먼저 스킨 정말 감사드립니다.

문의 관련해서는 아래 사진 참조 부탁드립니다.
글 업로드 후 썸네일이 아래와 같이 나오는 이슈가 있는데, 제가 무엇을 잘못했는지 찾지 못해 결국 문의를 드리게 되었습니다.
기존 방식과 동일하게 글을 업로드 하고 있었는데, 혹시 제가 어떻게 하면 해결할 수 있을지 아이디어가 있으시면 부탁드립니다.
해당 부분 링크는 아래와 같습니다. 감사합니다.
https://baramoss.tistory.com/category/IT/JavaScript

![image](https://user-images.githubusercontent.com/59471361/216196727-5a5c12c6-8920-4366-849c-2bf8fbb3b358.png)

**Comment by pronist** [comment, marked as answer]
@baramoss 구 버전을 사용하고 계시는 것 같습니다. 신 버전으로 바꾸시면 문제 없으실겁니다. v4.1 버전이 출시되었습니다.

**Comment by baramoss** [comment]
@pronist 신 버전으로 변경 후 썸네일이 정상 적용된 것 확인했습니다. 정말 감사드립니다!
